index.html

```diff
@@ -97,5 +97,208 @@ <h2 class="more_det">LET’S CONNECT</h2>
+      </div>
+      <ul class="work-detail">
+        <li class="canopy">CANOPY</li>
+        <li class="dev">Back End Dev</li>
+        <li class="year">2015</li>
+      </ul>
+      <article class="detail-cls">
+        <p>
+          A daily selection of privately<br />
+          personalized reads; no accounts or<br />
+          sign-ups required.
+        </p>
+      </article>
+      <div class="button-desc">
+        <h4 class="htm">html</h4>
+        <h4 class="css">css</h4>
+        <h4 class="javascript">javaScript</h4>
+      </div>
+      <button class="see-project">See Project</button>
+    </section>
+    <section class="work-card">
+      <div class="work_img2"></div>
+      <div class="name-pro">
+        <h3 class="work_name work-n">
+          Multi-Post<br />
+          Stories
+        </h3>
+      </div>
+      <ul class="work-detail">
+        <li class="canopy">CANOPY</li>
+        <li class="dev">Back End Dev</li>
+        <li class="year">2015</li>
+      </ul>
+      <article class="detail-cls">
+        <p>
+          A daily selection of privately<br />
+          personalized reads; no accounts or<br />
+          sign-ups required.
+        </p>
+      </article>
+      <div class="button-desc">
+        <h4 class="htm">html</h4>
+        <h4 class="css">css</h4>
+        <h4 class="javascript">javaScript</h4>
+      </div>
+      <button class="see-project">See Project</button>
+    </section>
+    <section class="work-card">
+      <div class="work_img3"></div>
+      <div class="name-pro">
+        <h3 class="work_name">Tonic</h3>
+      </div>
+      <ul class="work-detail">
+        <li class="canopy">CANOPY</li>
+        <li class="dev">Back End Dev</li>
+        <li class="year">2015</li>
+      </ul>
+      <article class="detail-cls">
+        <p>
+          A daily selection of privately<br />
+          personalized reads; no accounts or<br />
+          sign-ups required.
+        </p>
+      </article>
+      <div class="button-desc">
+        <h4 class="htm">html</h4>
+        <h4 class="css">css</h4>
+        <h4 class="javascript">javaScript</h4>
+      </div>
+      <button class="see-project">See Project</button>
+    </section>
+    <section class="work-card">
+      <div class="work_img4"></div>
+      <div class="name-pro">
+        <h3 class="work_name work-n">
+          Multi-Post<br />
+          Stories
+        </h3>
+      </div>
+      <ul class="work-detail">
+        <li class="canopy">CANOPY</li>
+        <li class="dev">Back End Dev</li>
+        <li class="year">2015</li>
+      </ul>
+      <article class="detail-cls">
+        <p>
+          A daily selection of privately<br />
+          personalized reads; no accounts or<br />
+          sign-ups required.
+        </p>
+      </article>
+      <div class="button-desc">
+        <h4 class="htm">html</h4>
+        <h4 class="css">css</h4>
+        <h4 class="javascript">javaScript</h4>
+      </div>
```

Comment:
- [ ] Kindly please make this `div` a `ul` and every child on the list make them a `li`.  That way, your code will be a lot more semantic and the browsers will have a better time understanding what you were trying to achieve 💪 
You can use this layout:

```HTML
<ul>
    <li>
    <li>
<ul>
```